.github/workflows/rust.yml

```diff
@@ -0,0 +1,76 @@
+
+name: Rust
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: RUSTFLAGS='-D warnings' cargo build --verbose --workspace
```

Comment:
It is unclear what we want to achieve with this stage given that borsh-rs is a library crate

Reply:
@frol This stage is for unit tests. Tests need to be build, and all their dependencies need to be downloaded from the internet. We can cut time of doing tests from 4m to 2m30s by using cache. 

Though, we don't ship many PR's, even if the `tests` takes 4m to finish, it's not that big of a deal.

Reply:
If that's what you want, `cargo build` requires `--tests` to actually build the tests. You could also use `cargo test --no-run` instead.

Reply:
> It is unclear what we want to achieve with this stage given that borsh-rs is a library crate

Oh, that's what you meant. Yes, we can remove cargo build.